packages/@react-aria/disclosure/docs/useDisclosure.mdx

```diff
@@ -0,0 +1,333 @@
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+
+---
+category: Navigation
+keywords: [disclosure, collapse, expand, aria]
+preRelease: alpha
+---
+
+# useDisclosure
+
+<PageDescription>{docs.exports.useDisclosure.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useDisclosure']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useDisclosure} links={docs.links} />
+
+## Features
+
+A disclosure is a collapsible section of content. It is composed of a trigger button and a panel that contains the content. `useDisclosure` can be used to implement these in an accessible way.
+
+* Support for mouse, touch, and keyboard interactions to open and close the disclosure
+* Support for disabled disclosures
+* Follows the disclosure ARIA pattern, semantically linking the trigger button and panel
+* Uses [hidden="until-found"](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden#the_hidden_until_found_state) in supported browsers
+
+## Anatomy
+
+{/* <Anatomy /> */}
+
+A disclosure consists of a trigger button and a panel. Clicking on or pressing <Keyboard>Enter</Keyboard> or <Keyboard>Space</Keyboard> while the trigger button is focused toggles the visibility of the panel. 
+
+`useDisclosure` returns props to spread onto the trigger button and panel.
+
+{/* <TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useDisclosure.return.id].properties} />
+</TypeContext.Provider> */}
+
+
+State is managed by the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useDisclosureState} />
+hook in `@react-stately/disclosure`. The state object should be passed as an option to `useDisclosure`.
+
+## Example
+
+This example displays a basic disclosure with a button that toggles the visibility of the panel.
+
+```tsx example export=true
+import {useDisclosureState} from '@react-stately/disclosure';
+import {useDisclosure} from '@react-aria/disclosure';
+import {useButton} from '@react-aria/button';
+
+function Disclosure(props) {
+  let state = useDisclosureState(props);
+  let panelRef = React.useRef<HTMLDivElement | null>(null);
+  let triggerRef = React.useRef<HTMLButtonElement | null>(null);
+  let {buttonProps: triggerProps, panelProps} = useDisclosure(props, state, panelRef);
+  let {buttonProps} = useButton(triggerProps, triggerRef);
+
+  return (
+    <div className="disclosure">
+      <h3>
+        <button className="trigger" {...buttonProps}>
+          <svg viewBox="0 0 24 24">
+            <path d="m8.25 4.5 7.5 7.5-7.5 7.5" />
+          </svg>
+          {props.title}
+        </button>
+      </h3>
+      <div className="panel" ref={panelRef} {...panelProps}>
+        <p>
+          {props.children}
+        </p>
+      </div>
+    </div>
+  );
+};
+
+<Disclosure title="System Requirements">
```

Comment:
i used `label` instead of `title` but i kinda like title more. might update the component docs so that there can be consistency between the two